### PR TITLE
Fixed email field error popup and event card css

### PR DESCRIPTION
**Pull request by paulpan05**

### CODE REVIEW

**Comment by StormFireFox1** [comment]
This PR fixes #299 and #301.

File: src/components/EventCard/styles.less [hunk 1]
```diff
@@ -13,80 +13,84 @@
...
 
-  img {
-    height: 150px;
-    margin: 0;
-    object-fit: cover;
-    width: 100%;
-  }
-
-  .info {
-    padding: 5% 10% 0 10%;
```
Comment (steets250) [inline src/components/EventCard/styles.less:41]: If you cut the top padding here, there should be enough room to squeeze 2-3 lines of title + include the bar at the bottom.

Comment (StormFireFox1) [reply src/components/EventCard/styles.less:41]: Removing the top padding makes the text be a bit too close to the event cover, but perhaps it's fine for a while.

**Review by StormFireFox1 — APPROVED** [review]
Looks good to me!

**Pull request merged**